cache/edge_lambdas/src/origin.ts

```diff
@@ -7,7 +7,7 @@ import {
 import { getRejection } from './rejection';
 
 export const request: CloudFrontRequestHandler = (event, context, callback) => {
-  const rejectResponse = getRejection(event);
+  const rejectResponse = getRejection();
```

Comment:
see `cache/edge_lambdas/src/rejection.ts`. `getRejection` does nothing with `event` and just returns `undefined`.